db-engine/src/main/java/com/splicemachine/db/impl/jdbc/TransactionResourceImpl.java

```diff
@@ -154,6 +155,7 @@
 							Properties info) throws SQLException 
 	{
 		this.driver = driver;
+		this.sessionProperties = info;
```

Comment:
Could we rename `info` to be `sessionProperties` (or even `sessionProps` or `sessionIinfo`) and use it later in `dbname = InternalDriver.getDatabaseName(url, info);`? I don't think the name difference is helpful.